op-batcher/batcher/driver.go

```diff
@@ -115,6 +122,13 @@ func NewBatchSubmitter(ctx context.Context, cfg Config, l log.Logger, m metrics.
 
 	cfg.metr = m
 
+	if cfg.BatchInboxAbi == nil {
+		biAbi, err := bindings.LeaderElectionBatchInboxMetaData.GetAbi()
+		if err == nil && biAbi != nil {
+			cfg.BatchInboxAbi = biAbi
+		}
```

Comment:
Should there be a `TODO` comment here or is it okay to continue with `BatchInboxAbi` unset?

Reply:
I'm not actually sure what should happen here. I need this to get the function selector for the contract, and it should always exist, but...

Reply:
If we crash if it's not set does that break anything?